dev.Dockerfile

```
@@ -0,0 +1,20 @@
+FROM python:3.11-slim-bullseye
```

```suggestion
FROM python:3.12-slim-bookworm
```
use newer python image